kolibri/core/exams/models.py

```diff
@@ -32,7 +32,19 @@ class Exam(AbstractFacilityDataModel):
     question_count = models.IntegerField()
 
     """
-    This field contains different values depending on the 'data_model_version' field.
+    The `question_sources` field contains different values depending on the 'data_model_version' field.
+
+    V2:
+        Similar to V1, but with a `counterInExercise` field
+        [
+            {
+                "exercise_id": <exercise_pk>,
+                "question_id": <item_id_within_exercise>,
+                "title": <exercise_title>,
+                "counterInExercise": <unique_count_for_question>
```

Comment:
should we make this `counter_in_exercise` for consistency?

Reply:
Yeah, let's do that. So to convert a V1 exam with `counterInExercise`,we'll just rename it to `counter_in_exercise`.